bin/makeReport.py

```diff
@@ -41,7 +41,7 @@
 nextclade['nextcladeVersion']=str(nextclade_version).strip()
 nextclade.set_index('program',inplace=True)
 n=nextclade.to_dict(orient='index')
-with open('nextclade.json','r') as inf:
+with open('nextclade.json','rt', encoding= 'utf-8') as inf:
```

Comment:
@JezSw do you recognize this change?

Reply:
This was introduced by Nick as part of 347803c6fd9f2f7f5c9f2790b037c370840139bc